linux/open_vm_tools/pre_ovt_install.yml

```diff
@@ -38,6 +38,50 @@
     - vmtools_is_installed
     - vmtools_install_type != 'package'
 
+- name: "Set fact of open-vm-tools install type"
```

Comment:
Can directly use linux_ovt_install_type, no need to add a new variable.

Reply:
No. `ovt_install_type` will be used by following test cases to set `vmtools_install_type`. We can't rely on the value of `linux_ovt_install_type` on an existing VM without any ovt installation testing.